src/error.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <Head>
+    <Title> {{ APP_TITLE_SHORT }} | Unexpected Error </Title>
+
+    <Meta
+      name="description"
+      :content="APP_DESCRIPTION"/>
+    <Meta
+      name="keywords"
+      :content="APP_KEYWORDS"/>
+    <Link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/favicon-apple-180.png"/>
+    <Link
+      rel="icon"
+      type="image/png"
+      sizes="512x512"
+      href="/favicon-512.png"/>
+    <Link
+      rel="icon"
+      type="image/png"
+      sizes="96x96"
+      href="/favicon-96.png"/>
+    <Link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/favicon-32.png"/>
+    <Meta
+      property="og:type"
+      content="website"/>
+    <Meta
+      property="og:title"
+      :content="APP_TITLE"/>
+    <Meta
+      property="og:description"
+      :content="APP_DESCRIPTION"/>
+    <Meta
+      property="og:image"
+      :content="`${APP_URL}/social-cover.png`"/>
+    <Meta
+      property="og:image:width"
+      content="1200"/>
+    <Meta
+      property="og:image:height"
+      content="600"/>
+    <Meta
+      property="og:image:alt"
+      :content="APP_TITLE"/>
+    <Meta
+      property="og:url"
+      :content="APP_URL"/>
+    <Meta
+      name="twitter:card"
+      content="summary_large_image"/>
+    <Meta
+      name="twitter:site"
+      :content="APP_CREATOR"/>
+    <Meta
+      name="twitter:creator"
+      :content="APP_CREATOR"/>
+  </Head>
+
+  <the-header/>
+  <div class="error">
+    <div class="error__parallax">
+      <page-header>
+        Unexpected Error
+      </page-header>
+
+      <app-panel class="error__panel">
+        <h1 class="error__heading">
```

Comment:
Now that I see it live I would change it to h2, to me it looks too big and also does not have a hierarchy.
If you agree let's also add a comment to the design.

Reply:
fixed

Reply:
![image](https://github.com/aeternity/aescan/assets/15363559/ba5e79be-a5cb-4f45-a81e-7db90b7aca7f)